### DISCUSSION TITLE
github student pack

### DISCUSSION THREAD

**Discussion by AbdullahWaqarSatti** [community/community#68298] [New to GitHub]
What is github student pack

**Comment by milindaShehan** [comment]
The GitHub Student Developer Pack is a collection of resources and tools that are made available to students at no cost. It's designed to help students learn and develop their coding skills. The pack includes a variety of benefits from GitHub and other partner services.
[check out this](https://education.github.com/pack)

**Comment by Nazirkhanwazir** [comment]
to avail student package you had to verify yourself as student from github and then there are lot of apps and websites which you can use for free for period of time.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
